docs/api-readme.md

```diff
@@ -0,0 +1,35 @@
+# LitElement API Documentation
+
+## Install @polymer/lit-element
+
+```
+npm install --save @polymer/lit-element
+```
+
+## Modules
+
+### lit-element
+
+[lit-element](/api/modules/_lit_element_.html)
+
+Contains LitElement base class. Import `LitElement` and `html` from this module to create a component:
+
+```js
+import { LitElement, html } from '@polymer/lit-element';
+
+class MyElement extends LitElement {
+  render() {
+      return html`<p>your template here</p>`;
+  }
+}
+
+customElements.define('my-element', MyElement);
+```
+
+Inherits API functionality from [updating-element](#updating-element).
+
+### updating-element
+
+[updating-element](/api/modules/_lib_updating_element_.html)
+
+Provides API functionality for LitElement. Manages properties and attributes; Performs element updates.
```

Comment:
"Provides API functionality for LitElement." isn't very clear to me, especially why it is separate from LitElement. I'd try something like:

"Custom Element base class that supports declaring observable properties, reflecting attribute to properties, and the core update lifecycle methods"

Reply:
Why is it separate from LitElement?

Reply:
Because all those things don't have any dependency on lit-html, so they could be used with another templating system, or none at all. We'll probably split it out into a separate package at some point.

Reply:
cool... will that be obvious to the audience?

Reply:
I think it would be more obvious if LitElement was described as an extension of UpdatingElement that ties it into lit-html by letting users define a render() method that's called at the right time in update().

Reply:
> o they could be used with another templating system

For the record:

https://github.com/blikblum/snabbdom-element (uses snabbdom virtual dom)
https://github.com/blikblum/raw-element (no templating at all)